packages/sdk-client/src/api/api-client.ts

```diff
@@ -5,8 +5,13 @@ import FormData = require('form-data');
 
 export enum PaginationEnum {
   NONE,
+  /** Used by the Numbers API */
   TOKEN,
+  /** used by the SMS API */
   PAGE,
+  /** used by the Elastic SIP Trunking API */
+  PAGE2,
+  /** used by the Fax API */
   PAGE3
```

Comment:
What can we tell from the names of the enum values? Maybe we can add descriptive name to not get confused later? e.g.
```suggestion
  INDEX_SMS_PAGE,
  /** used by the Elastic SIP Trunking API */
  ELASTIC_SIP_INDEX_PAGE,
  /** used by the Fax API */
  FAX_INDEX_PAGE
```

Reply:
I hope that not all the APIs will use a different pagination and that the same pagination type can be used by several APIs as the OAuth2 authentication is used by several APIs

Reply:
Still it would nice to have a meaningful names to the enum then to clarify intention but that's up to you